src/main/java/apoc/text/Strings.java

```diff
@@ -372,4 +372,14 @@ public String base64Decode(@Name("text") String text) {
         byte[] decoded = Base64.getDecoder().decode(text.getBytes());
         return new String(decoded);
     }
+
+    @UserFunction
+    @Description("apoc.text.charAt(text, index) - the hex value string of the character at the given index")
+    public String charAt(@Name("text") String text, @Name("index") Long index) {
```

Comment:
shouldn't this return the long/int value?
and then a separate function do the toHex ?

Reply:
Could do that. Ran into some issues with `Character.getNumericValue()` not returning the correct values for certain characters, I'll check for alternatives.